src/main/java/nl/pjiwm/kaizominecraft/Kaizo.java

```diff
@@ -10,15 +11,10 @@
     @Override
     public void onEnable() {
         // Plugin startup logic
+        CustomWorldManager.geneRateWorlds();
```

Comment:
Just a question, is it **actually** "geneRate"? Because if so, I hate Spigot/Bukkit